src/resources/views/pages/dashboard/post-footer.blade.php

```diff
@@ -1,113 +1,99 @@
-                                                @if ($comment->user_id == auth()->id())
-                                                    <span
-                                                        class="text-sm font-semibold ml-1 cursor-pointer comment-edit-link"
-                                                        onclick="editComment({{ $comment->id }})">
-                                                        Edit
-                                                    </span>
-                                                @endif
+                                    <li class="text-sm comment-content">{{ $comment->content }}</li>
+                                    <div class="flex items-center gap-2 text-sm font-thin comment-actions">
+                                    <span class="comment-time">{{ $comment->formatted_time }}</span>
+                                        @if ($comment->user_id == auth()->id() || $post->user_id == auth()->id())
+                                            @include('pages.forms.delete-comment-form')
+                                            @if ($comment->user_id == auth()->id())
+                                                <span class="text-sm font-semibold ml-1 cursor-pointer comment-edit-link"
+                                                    onclick="editComment({{ $comment->id }})">
+                                                    Edit
+                                                </span>
                                             @endif
-                                        </div>
-                                        <!-- Edit Comment Form -->
-                                        @include('pages.forms.edit-comment-form')
+                                        @endif
                                     </div>
+                                    @include('pages.forms.edit-comment-form')
                                 </div>
                             </div>
-                        @endforeach
-                    </ul>
-                </div>
-                <div id="comment{{ $post->id }}"
-                    class="d-dashboard__border-gray mb-2 flex p-2 items-center w-full hidden transition-all duration-300">
-                    <div class="flex flex-col h-max-64 w-full">
-                        <div>
-                            <!-- Add Comment Form -->
-                            @include('pages.forms.add-comment-form')
                         </div>
-                    </div>
-                </div>
+            @endforeach
+        </ul>
+    </div>
+    <div id="comment{{ $post->id }}"
+        class="d-dashboard__border-gray mb-2 flex p-2 items-center w-full hidden transition-all duration-300">
+        <div class="flex flex-col h-max-64 w-full">
+            <div>
+                @include('pages.forms.add-comment-form')
+            </div>
+        </div>
+    </div>
+    
```

Comment:
I think theres an extra spaces to this line, please remove it.

<img width="354" alt="Screenshot 2024-06-08 at 4 05 31 PM" src="https://github.com/xReddskie/microblog-ojt2/assets/144191092/c98a8114-2d42-4d3e-a983-8ec140c550fe">